### PR TITLE
Adding a doc to compare fabrikate against HelmOperator

### PR DESCRIPTION
**Pull request by mtarng**

### CODE REVIEW

File: docs/comparisons.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+| [Fabrikate](https://github.com/microsoft/fabrikate) | [Helm Operator](https://github.com/fluxcd/helm-operator) | [Kustomize](https://github.com/kubernetes-sigs/kustomize) |
+|:--|:--|:--|
+| Fabrikate can manages both static Kubernetes manifests or Helm charts by wrapping them into [Components](https://github.com/microsoft/fabrikate/blob/develop/docs/component.md). Helm charts in a component can have their values set via config .yaml or .json values that fabrikate will then generate static manifests. <br><br> A single fabrikate component can be mapped to multiple config files so that different sets of releases can be created and managed. <br><br> Multiple components can be combined into a [High Level Definitions](https://github.com/microsoft/bedrock/blob/master/docs/gitops-pipeline.md#deep-dive-high-level-definitions) for ease of management and repeatability. | The Helm Operator makes use of Helm Release CRDs to manage releases and overwrite default values of a helm chart. Multiple Helm Releases can point to the same Helm Chart and parameterize the values for each release. | N/A |
```
Comment (edaena) [inline docs/comparisons.md:18]: can manages -> can manage or manages

File: docs/comparisons.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+| [Fabrikate](https://github.com/microsoft/fabrikate) | [Helm Operator](https://github.com/fluxcd/helm-operator) | [Kustomize](https://github.com/kubernetes-sigs/kustomize) |
+|:--|:--|:--|
+| Once an application image is built and published to a container repository, then the associated config values in the Component/HLD repository needs to have the image.tag value updated to the latest version, either manually or as an automated step after publishing the image in a CI/CD pipeline. Once the version is set, then the fabrikate generation pipeline should be triggered to generate the latest manifests with the updated image tag. Once the final manifests are generated, the GitOps operator should sync the changes onto the cluster. | Once an application image is built and published to a container repository, then the associated Helm Release needs to have the image.tag value updated to the latest version, either manually or as an automated step after publishing the image in a CI/CD pipeline. Once the Helm Release has the latest values, then the Helm Operator will deploy the latest image to the repository. | N/A |
```
Comment (edaena) [inline docs/comparisons.md:30]: "then the associated config values in the component/hld repository needs to have the image.tag value updated..." -> the config value `image.tag` in the Component/HLD repository needs to be updated with the latest version, either manually....

File: docs/comparisons.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+|:--|:--|:--|
+| The chosen GitOps operator needs to be added to the cluster and configured to the final generated manifest repository. <br><br> The [bedrock](https://github.com/microsoft/bedrock) and [bedrock-cli](https://github.com/microsoft/bedrock-cli) projects can be leveraged to configure clusters as well as supporting pipelines in an automated and repeatable pattern. | The Flux operator and Helm Operator must be installed onto a cluster then configured to the appointed Helm Release repository.| N/A |
+
+### Helm version support
+| [Fabrikate](https://github.com/microsoft/fabrikate) | [Helm Operator](https://github.com/fluxcd/helm-operator) | [Kustomize](https://github.com/kubernetes-sigs/kustomize) |
+|:--|:--|:--|
+| Both Helm 2 and 3 are supported. <br><br> **Note** The latest releases of fabrikate support Helm v3. To use fabrikate with Helm2, either use an older release (v__ and older) or build a binary off the [helm2](https://github.com/microsoft/fabrikate/tree/helm2) branch. | Both Helm 2 and 3 are supported | N/A |
```
Comment (evanlouie) [inline docs/comparisons.md:51]: Helm vs Fab versions outlined here: https://github.com/microsoft/fabrikate/wiki/Roadmap-Updates#release-v1-and-v2

**Comment by andrebriggs** [comment]
@mtarng Instead of N/A let's put TODO in the Kustomize column

**Review by andrebriggs — APPROVED** [review]

**Pull request merged**